### PR TITLE
Enable dev line builds of less

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**

### CODE REVIEW

**Review by MikeFultonDev — COMMENTED** [review]
If there is no rush it would seem better to get tests running even if they have a high failure rate than skipping tests

File: basictest.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+actual=$(./less --version | head -1)
+expected='less 6.* (POSIX regular expressions)'
```
Comment (AnthonyGiorgio) [inline basictest.sh:3]: This is going to break when v700 is released.  Can it be more generic?

**Review by AnthonyGiorgio — APPROVED** [review]
Looks good, but needs a tweak to the test regex pattern.

**Comment by IgorTodorovskiIBM** [comment]
@MikeFultonDev @AnthonyGiorgio this is in preparation for upstreaming less since it's one of the core tools. They accept pull requests here: https://github.com/gwsw/less/pulls

@AnthonyGiorgio let me know if you prefer to take that on, that would be the next step.

**Comment by AnthonyGiorgio** [comment]
> @MikeFultonDev @AnthonyGiorgio this is in preparation for upstreaming less since it's one of the core tools. They accept pull requests here: https://github.com/gwsw/less/pulls
> 
> @AnthonyGiorgio let me know if you prefer to take that on, that would be the next step.

Is that test from the less repo?  Or is it something that we cooked up ourselves?

**Comment by IgorTodorovskiIBM** [comment]
Added licenses as well

File: basictest.sh (hunk 1 above)
Comment (IgorTodorovskiIBM) [reply basictest.sh]: fixed

**Pull request merged**